src/main/java/com/vmware/vipclient/i18n/messages/api/opt/server/ComponentBasedOpt.java

```diff
@@ -65,7 +65,10 @@ public void getComponentMessages(MessageCacheItem cacheItem) {
 		        JSONObject respObj = (JSONObject) JSONValue.parse((String) response.get(URLUtils.BODY));
 		        try {
 	        		if (getResponseCode(respObj) == 200 && 
-	        				// Do not use service response if not matching the locale in the request
+	        				// If the response is 200, there are 2 possible cases:
+	        				// 1. the response's locale matches the request's locale OR
+	        				// 2. the response's locale is for the default locale (service's fall back behavior)
```

Comment:
Service is removing fallback behavior.

Reply:
We don't need to consider fallback because service won't fallback in the future.